raspicam/classify_capture.py

```diff
@@ -14,14 +14,59 @@
 
 """A demo to classify Raspberry Pi camera stream."""
 import argparse
-import os
-import io
-import time
+import collections
 from collections import deque
+import io
 import numpy as np
+import operator
+import os
 import picamera
+import tflite_runtime.interpreter as tflite
+import time
+
+Class = collections.namedtuple('Class', ['id', 'score'])
+EDGETPU_SHARED_LIB = 'libedgetpu.so.1'
+
+def make_interpreter(model_file):
+    model_file, *device = model_file.split('@')
+    return tflite.Interpreter(
+      model_path=model_file,
+      experimental_delegates=[
+          tflite.load_delegate(EDGETPU_SHARED_LIB,
+                               {'device': device[0]} if device else {})
+      ])
+
+def input_size(interpreter):
```

Comment:
Even though there is only one example here i'd still factor these helpers into a common.py.

Ideally the common.py would be identical between the different sub-example-subfolders but i dont see an easy way
to share just one file without making the import more complicated.